src/PokeList.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import PokeItem from './PokeItem.js';
+
+class PokeList extends Component {
+    state = {  }
+    render() { 
+        return (
+            <article className='pokemon'>
```

Comment:
Great job on trying different semantic tags! In this case, I would actually use a <section> for the list and then you can use <article> for each individual PokeItem.